### DISCUSSION TITLE
1.0.9a

### DISCUSSION THREAD

**Discussion by saysora** [saysora/p5#11] [General]
## Theme Fix Part 1 (1.0.9a)
Thanks Discord.

This update fixes (for now)
* User messaging area
* User messaging text color
* User messaging button hover color
* Channel messages fonts
* Channel messages interaction buttons
* Discord buttons for mic, deafen, and settings

Much more to come.

### NOTE: There will be a discussion created for this update, if you find more areas or these aren't working, you are free to use that discussion to point out flaws / more needed fixes.

<hr /><em>This discussion was created from the release <a href='https://github.com/saysora/p5/releases/tag/1.0.9a'>1.0.9a</a>.</em>

**Comment by miszerkyst** [comment]
Not really a flaw or a fix, but something that would make this theme really work for me at least, and potentially others, is being able to set a background! I love the theme but would also love my boi Yusuke as a background on it.
